aiidalab_widgets_base/computational_resources.py

```diff
@@ -884,14 +884,18 @@ def _configure_computer_local(self, computer: orm.Computer, user: orm.User):
 
     def _run_callbacks_if_computer_exists(self, label):
         """Run things on an existing computer"""
-        try:
-            orm.Computer.objects.get(label=label)
+        if self._computer_exists(label):
             for function in self._on_setup_computer_success:
                 function()
+            return True
+        return False
+
+    def _computer_exists(self, label):
+        try:
+            orm.Computer.objects.get(label=label)
```

Comment:
Any reason why not use `aiida.orm.load_computer()`? Seems more understandable to me.

```suggestion
            orm.load_computer(label=label)
```

Reply:
`orm.load_computer` is better, I agree.